src/visitpy/visit_utils/src/encoding.py

```diff
@@ -54,7 +54,7 @@ def extract(ifile,opattern):
     Requires opattern to use printf style # format like "file%04d.png".
     """
     enc_bin = ffmpeg_bin()
-    if not ffmpeg_bin is None:
+    if not enc_bin is None:
         cmd = "%s -i %s -f image2 %s" % (enc_bin,ifile,opattern)
```

Comment:
thanks for fixing this, i assume this now gives us a much better error message.

Reply:
Yes!
```
    visit_utils.encoding.extract("test_0.mpg", "frame%04d.png")
  File "/usr/workspace/wsa/brugger/visit_dev_git/visit/build2/lib/site-packages/visit_utils/encoding.py", line 61, in extract
    raise VisItException("ffmpeg not found: Unable to extract frames.")
visit_utils.common.VisItException: <VisIt Error> ffmpeg not found: Unable to extract frames.
>>> 
! ERROR: Missing Exit() when running test file: hybrid/movie.py
[Test suite run complete @ 2023:03:13:16:39:41 (wall time = 56)]
!! Test suite run finished with 1 error.
[Cleanup: Waiting for delayed writes]
[Cleanup: Removing _run directory]
```